gendiff/string_templates.py

```diff
@@ -0,0 +1,148 @@
+from typing import Any, Dict
+
+from gendiff.diff import get_diff_view
+
+diffs_templates = {
+    "missing": "{indent}- {key}:{symb}{value}\n",
+    "added": "{indent}+ {key}:{symb}{value}\n",
+    "changed": "{indent}- {key}:{symb}{value}\n{indent}+ {key}: {ch_value}\n",
```

Comment:
what is the purpose of `symb`?

Reply:
the reason is choice between -/+ depending on dict1[key] or dict2[key] is nested dict